modules/platforms/cpp/binary/src/impl/binary/binary_object_impl.cpp

```diff
@@ -40,6 +40,20 @@ namespace ignite
             {
                 if (idRslvr)
                     this->idRslvr = idRslvr->Clone();
+
+                while (true)
```

Comment:
Can you please explain this loop logic? Our BinaryObject format does not require looping like this, looks confusing.